### DISCUSSION TITLE
Comparison between main repo and fork

### DISCUSSION THREAD

**Discussion by joulev** [community/community#3730] [Mobile] [answered]
Basically, adding "This branch is x commits ahead abc:main" like in the web version.

**Comment by brianlovin** [comment, marked as answer]
Thanks for the suggestion @joulev! Agreed this would be nice context to add to the branch switcher and code browser UI. I'll spin up a tracking issue for our team, but not sure exactly when this will be built 😊

**Reply by linhzz999** [reply]
Ok

**Comment by linhzz999** [comment]
Ok

**Comment by TPS** [comment]
While doing this, also please bring back ability to navigate to main/parent repo _from_ the fork, which was lost around the time forking a repo was introduced to the app.
